terraform/variables.tf

```diff
@@ -37,3 +37,11 @@ variable "db_password" {
 variable "ssh_public_key" {
   default = "~/.ssh/id_rsa.pub"
 }
+
```

Comment:
I am failing to see where/how this file (variables.tf) is used. It looks to me like we read all vars from the JSON config and pass them explicitly via the command line. Does terraform make use of this implicitly somehow?

Reply:
> Does terraform make use of this implicitly somehow?

Correct. There are some more *.tf files. Terraform reads all of them.

Reply:
So do we really need to have defaults in two places (JSON config and this variables file)? They are going to get replaced I imagine. I think it's a bit confusing, maybe we can blank all the variables we end up reading through the JSON config.

Reply:
That's a fair point.

Reply:
Done